README.md

```diff
@@ -84,8 +84,8 @@ coefficients, each of which is smaller than *t*.
 To encrypt *m* with key *s*, select a polynomial *e* from the distribution *Y*
 and a polynomial *a* from the uniform distribution (each coefficient of *a* is
 chosen uniformly at random). *e* is a random amount of error that makes it
-difficult to extract the error from the ciphertext. Both *a* and *e* are nonces
-that should be regenerated freshly for each encryption.
+difficult to extract the key from the ciphertext. Both *a* and *e* should be
+regenerated freshly for each encryption.
```

Comment:
1. You might mean "key" instead of "error".
2. Somewhat opinionated: if by "nonce" you mean "number used once", then I don't think it's fair to call these nonces, as it's not sufficient for them to be non-repeating.